### PR TITLE
Merging 'tensorboard-integration' with 'master'

### PR DESCRIPTION
**Pull request by kadhirumasankar**
Added a function to `logger.py` which logs metrics (namely model score, the best model score, EMA score, the best EMA score, and training loss) to TensorBoard files inside `~/runs/uav-classifier/`

TensorBoard can be run using `tensorboard --logdir=~/runs/uav-classifier/`

### CODE REVIEW

File: hawk_eye/configs/vovnet.yaml [hunk 1]
```diff
@@ -14,5 +14,5 @@ training:
     end_lr: 1.0e-8
     warmup_fraction: 0.05
   epochs: 15
-  batch_size: 200
+  batch_size: 1
```
Comment (kadhirumasankar) [inline hawk_eye/configs/vovnet.yaml]: Need to change this back

File: hawk_eye/train/train_clf.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 import os
 import yaml
 
+
```
Comment (kadhirumasankar) [inline hawk_eye/train/train_clf.py]: ↪️

**Review by kadhirumasankar — COMMENTED** [review]
👍

File: hawk_eye/train/train_utils/logger.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import pathlib
 import logging
+import torch.utils.tensorboard as tensorboard
```
Comment (alexwitt23) [inline hawk_eye/train/train_utils/logger.py]: Leave empty line here:
```
system

external

project
```

**Review by alexwitt23 — APPROVED** [review]
Nice work to all involved! Small change requested withe imports, and can you add the loss to the log, too?

**Review by alexwitt23 — CHANGES_REQUESTED** [review]
Sorry, meant to request changes on the review above

**Comment by kadhirumasankar** [comment]
> Nice work to all involved! Small change requested withe imports, and can you add the loss to the log, too?

@alexwitt2399 training loss is already being logged with `log.info` and `log.metric` (i.e. to TensorBoard). What do I need to change?

**Comment by alexwitt23** [comment]
I see, I missed that line. LGTM

**Review by alexwitt23 — APPROVED** [review]
